docs/developers/bridges/usdcbridge.md

```diff
@@ -0,0 +1,169 @@
+---
+title: USDC Bridge
+sidebar_position: 2
+---
+
+# Introduction
+
+### The Circle USDC Bridge on Linea
+
+This article is about a crypto-asset bridge that has been deployed to Linea. It has been designed and built by [ConsenSys](https://consensys.net/), to the specifications of [Circle](https://www.circle.com/en/), for the purposes of bridging [USDC](https://www.circle.com/en/usdc) between networks: initially, between the [Goerli Ethereum Testnet](https://goerli.net/) and the [Linea Goerli Testnet](https://linea.build/).
+
+### What does it do?
+
+Circle mints USDC natively on each network on which the currency is available. Therefore, this could be considered a “burn and mint” bridge: the USDC submitted for bridging on the chain of origin is burned, and the equivalent amount of USDC is minted on Linea.
```

Comment:
It's worth mentioning why we are building this bridge instead of using the canonical token bridge.
The USDC bridge includes mechanics that will allow Circle to take over and issue canonical USDC on Linea (instead of  a bridged version).